src/components/Person.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const Person = ({
+  name,
+  age,
+  sex,
+  isMarried,
+  partnerName,
+}) => {
+  const sexOfPartner = sex === 'm'
+    ? 'wife'
+    : 'husband';
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">{`My name is ${name}`}</h2>
+      <p className="Person__age">{age ? `I am ${age}` : ''}</p>
```

Comment:
```suggestion
      {age && (
        <p className="Person__age">
          I am {age}
        </p>
      )}
```
Now you are rendering an empty `p` if there's no `age` value. Let's not render it at all in this case